example/tutorial_vgg16.py

```diff
@@ -174,6 +140,10 @@ def fc_layers(net):
 
 img1 = imread('data/laska.png', mode='RGB')  # test data in github
 img1 = imresize(img1, (224, 224))
+# rescale pixels values in the range of 0-1
+img1 = img1/255.0
+if ((0 <= img1).all() and (img1 <= 1.0).all()) is False:
```

Comment:
it seems you did not x255 inside the vgg model.

Reply:
Yes, in this demo, we just deal with the range of input image. It seem as the tutorial vgg19. Actually, the input value will also be [0,255]